### PR TITLE
adapt rpm spec file for 2.0.36, test on CentOS7

### PR DESCRIPTION
**Pull request by marcofl**

### CODE REVIEW

File: rhel/SPECS/burp.spec [hunk 1]
```diff
@@ -1,10 +1,10 @@
 Name:		burp
 Summary:	Burp is a network-based simple yet powerful backup and restore program for Unix and Windows.
-Version:	2.0.28
-Release:	4%{?dist}
+Version:	2.0.36
+Release:	1%{?dist}
 License:	GPL
 URL:		http://burp.grke.org/
-Source0:	https://github.com/grke/burp/archive/burp-%{version}.tar.bz2
+Source0:	https://github.com/grke/burp/archive/burp-%{version}.zip
```
Comment (rubenk) [inline rhel/SPECS/burp.spec]: What's wrong with tar.gz?

File: rhel/SPECS/burp.spec [hunk 2]
```diff
@@ -23,12 +23,13 @@ backing up Windows computers.
 %setup -q -n %{name}-%{version}
 
 %build
+autoreconf -vif
```
Comment (rubenk) [inline rhel/SPECS/burp.spec]: You'll need to BuildRequire autoconf, automake, libtool and pkg-config if you do this.

File: rhel/SPECS/burp.spec [hunk 3]
```diff
@@ -23,12 +23,13 @@ backing up Windows computers.
...
 %configure --sysconfdir=%{_sysconfdir}/%{name}
 make %{?_smp_mflags}
 
 %install
-rm -rf %{buildroot}
-make install DESTDIR=%{buildroot}
+rm -rf %{buildroot}
```
Comment (rubenk) [inline rhel/SPECS/burp.spec]: Trailing whitespace.

File: rhel/SPECS/burp.spec [hunk 4]
```diff
@@ -38,7 +39,7 @@ install -p -m 0644 %{SOURCE2} %{buildroot}%{_unitdir}/
 %endif
 
 %clean
-rm -rf %{buildroot}
+rm -rf %{buildroot}
```
Comment (rubenk) [inline rhel/SPECS/burp.spec]: The same here.

**Comment by marcofl** [comment]
@rubenk thank you, done. ok?

**Comment by rubenk** [comment]
LGTM :+1:

**Comment by grke** [comment]
Thank you both. :)

**Pull request merged**